scratch/benchmarks/benchmark_fgpu.py

```diff
@@ -390,13 +426,14 @@ async def main():  # noqa: D103
     parser.add_argument("--servers", type=str, default="servers.toml", help="Server description file [%(default)s]")
     parser.add_argument("--dsim-server", type=str, default="dsim", help="Server on which to run dsims [%(default)s]")
     parser.add_argument("--fgpu-server", type=str, default="fgpu", help="Server on which to run fgpu [%(default)s]")
-    parser.add_argument("-v", "--verbose", action="store_true", help="Emit stdout/stderr [no]")
+    parser.add_argument("-v", "--verbose", action="count", default=0, help="Increase verbosity [no]")
     parser.add_argument(
         "--calibrate", action="store_true", help="Run at multiple rates to calibrate expectations [%(default)s]"
```

Comment:
What is the difference between calibration and straight search? When you say that you expect to take 100 hours to do the search, how long does the calibration step take? I'm guessing 40-60 rates * 100 trials per rate * 1 minute = also about 100 hours.

Reply:
The 100 hours is for the calibration, which produces the calibrate-n*.txt files. The actual search is capped at 40 trials by default (the actual number depends on the `--interval` parameter controlling how tight a confidence interval you want).